datahub-web-react/src/app/glossary/GlossaryEntitiesList.tsx

```diff
@@ -1,44 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Empty } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+
 import styled from 'styled-components/macro';
 import { GlossaryNodeFragment } from '../../graphql/fragments.generated';
-import { GlossaryNode, GlossaryTerm } from '../../types.generated';
+import { EntityType, GlossaryNode, GlossaryTerm } from '../../types.generated';
 import { useEntityRegistry } from '../useEntityRegistry';
 import GlossaryEntityItem from './GlossaryEntityItem';
+import StyledButton from '../entity/shared/components/styled/StyledButton';
+import CreateGlossaryEntityModal from '../entity/shared/EntityDropdown/CreateGlossaryEntityModal';
 
 const EntitiesWrapper = styled.div`
     flex: 1;
     overflow: auto;
     padding-bottom: 20px;
 `;
 
+const ButtonContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`;
+
 interface Props {
     nodes: (GlossaryNode | GlossaryNodeFragment)[];
     terms: GlossaryTerm[];
+    refetchForTerms?: () => void;
+    refetchForNodes?: () => void;
 }
 
 function GlossaryEntitiesList(props: Props) {
-    const { nodes, terms } = props;
+    const { nodes, terms, refetchForTerms, refetchForNodes } = props;
     const entityRegistry = useEntityRegistry();
+    const [isCreateTermModalVisible, setIsCreateTermModalVisible] = useState(false);
+    const [isCreateNodeModalVisible, setIsCreateNodeModalVisible] = useState(false);
+
+    const contentsData =
+        nodes.length === 0 && terms.length === 0 ? (
```

Comment:
i find this ternary kind of hard to read / follow. How do you feel about doing it a little simpler:
```
if (!nodes.length && !terms.length) {
    return ( ... empty content
}

return ( ... not empty content
```

Reply:
Worked on that and committed my changes.